src/app/shared/custom-components/toolbar/search/search.component.html

```diff
@@ -1,11 +1,14 @@
-<form class="example-form">
+<form class="search-form">
```

Comment:
Seems better create container for input and button and we can skip absolute positioning
>div class="search-form__field>
    >button class="search-form__button" mat-icon-button>  //we can use here "button button_c_blue" after extracting from css
      >mat-icon class="icon icon_s">search>/mat-icon>
    >/button>
   > input
      type="text"
      class="search-form__input"
      placeholder="Search"
      [formControl]="control"
      [matAutocomplete]="auto"
    />
  >/div>